authors/apps/authentication/views.py

```diff
@@ -26,6 +27,20 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
 
+        # Set all variables to be used with send_email function
+        subject = "Welcome to Authors Haven"
+        contact_message = "To {},".format(serializer.data.get('username')) +\
+            "\n Thank you for joining Authors Haven. " +\
+            "We are glad to have you on board. " +\
+            "Please use the link http://127.0.0.1:8000/api/users/login/" +\
```

Comment:
The login URL is being passed directly. It won't work on production

Reply:
> The login URL is being passed directly with the localhost as the domain. It won't work on production. You could give the production url

I have changed it. You can now view a different URL depending on the environment